beamer/tests/contracts/test_request_manager_fees.py

```diff
@@ -246,16 +246,73 @@ def test_different_fees(request_manager, token, claim_period, claim_stake):
     assert token.balanceOf(claimer) == amount * 2 + lp_fee_1 + lp_fee_2
 
 
-def test_ppm_out_of_bound(request_manager, token):
+def test_ppm_out_of_bound(request_manager):
     ppm = 999_999
 
-    request_manager.updateToken(token.address, 0, 0, ppm, 0)
-    request_manager.updateToken(token.address, 0, 0, 0, ppm)
+    request_manager.updateFees(0, ppm, ppm)
 
     ppm = 1_000_000
 
     with ape.reverts("Maximum PPM of 999999 exceeded"):
-        request_manager.updateToken(token.address, 0, 0, ppm, 0)
+        request_manager.updateFees(0, ppm, 0)
 
     with ape.reverts("Maximum PPM of 999999 exceeded"):
-        request_manager.updateToken(token.address, 0, 0, 0, ppm)
+        request_manager.updateFees(0, 0, ppm)
+
+
+@pytest.mark.parametrize("transfer_cost", [int(400e12)])
```

Comment:
i think we didnt use this right ?

Reply:
it is used by other fixtures (it will be used during deployment)